### PR TITLE
PRS in EMML1832 (no.8)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Various PRS in additions in EMML1832.

### CODE REVIEW

File: new/PRS14287Mangaday.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">መንገደይ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Mangaday</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">qamaẓānay (= liqa maẓāne)</occupation>
```
Comment (thea-m) [inline new/PRS14287Mangaday.xml:51]: is ẓ used here for ḍ (which has to be used according to the guidelines)?

File: new/PRS14287Mangaday.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (thea-m) [inline new/PRS14287Mangaday.xml:35]: "t:...xmlns:t="http://www.tei-c.org/ns/1.0"" sneaked in again...could you delete it?

File: new/PRS14289Matewos.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <revisionDesc>
+            <change who="CH" when="2024-01-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Mātewos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (thea-m) [inline new/PRS14289Matewos.xml:49]: filda missing and to be swapped

File: new/PRS14294Zakkaryas.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘካርያስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zakkārǝyās</persName>
+                    <floruit notBefore="1300" notAfter="1350">First half of 14th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="snap:UncleOf" active="PRS14294Zakkaryas" passive="PRS00000000000000"/>
```
Comment (thea-m) [inline new/PRS14294Zakkaryas.xml:53]: record still missing?

File: new/PRS14296Kiros.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1"></persName>
```
Comment (thea-m) [inline new/PRS14296Kiros.xml:48]: fidal missing

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**